src/main/java/frc/robot/systems/DriveFSMSystem.java

```diff
@@ -166,27 +137,23 @@ public void resetTeleop() {
 
 		currentState = FSMState.TELEOP_STATE;
 
-		timer.reset();
-		timer.start();
+		stateTimer.reset();
+		stateTimer.start();
 
 		// Call one tick of update to ensure outputs reflect start state
 		update(null);
 	}
+
 	/**
 	 * Update FSM based on new inputs. This function only calls the FSM state
 	 * specific handlers.
 	 * @param input Global TeleopInput if robot in teleop mode or null if
 	 *        the robot is in autonomous mode.
 	 */
 	public void update(TeleopInput input) {
-		double updatedTime = timer.get();
+		double updatedTime = stateTimer.get();
```

Comment:
updatedTime isn't used anywhere else, and currentTime can be replaced by calls to stateTimer.get().